eq-author/src/App/page/Design/answers/MultipleChoiceAnswer/MultipleChoiceAnswer.test.js

```diff
@@ -103,6 +109,44 @@ describe("MultipleChoiceAnswer", () => {
     });
   });
 
+  it("should render the Options", () => {
+    const extraProps = {
+      canMoveDown: false,
+      canMoveUp: false,
+      onMoveUp: jest.fn(),
+      onMoveDown: jest.fn(),
+    };
+
+    const Options = wrapper.find(Reorder).renderProp("children")(
+      extraProps,
+      option
+    );
+
+    expect(Options.find(Option)).toHaveLength(1);
+  });
+
+  it("should render the Options with additional answer", () => {
+    const extraProps = {
+      canMoveDown: false,
+      canMoveUp: false,
+      onMoveUp: jest.fn(),
+      onMoveDown: jest.fn(),
+    };
+
+    option.additionalAnswer = {
+      id: "add1",
+    };
+
+    const Options = wrapper.find(Reorder).renderProp("children")(
+      extraProps,
+      option
+    );
+
+    expect(
+      Options.find("MultipleChoiceAnswer__SpecialOptionWrapper")
```

Comment:
Is this looking for the className? If so, would it be better to look for something more specific?

Reply:
No, MultipleChoiceAnswer__SpecialOptionWrapper is the generated tagname. You can check the generated snapshot to see the tags

Reply:
> Looks very and works perfectly.
> 
> Also, I was thinking this snapshot `eq-author/src/App/page/Design/answers/MultipleChoiceAnswer/__snapshots__/Option.test.js.snap` is rather large and hard to review... Is there any changes that can be made so we are not relying too much on it?

I haven't introduced any new snapshots, just updated existing ones to match the code changes. If this is an issue then I think it should be addressed in a separate ticket.

Reply:
Might be nice to start pulling away from snapshots, they tend to be a pain to manage and clog up the reviews. Think we could do with discussing whether we want to stop creating new enzyme snapshot tests going forwards and instead slowly supplant them with rtl tests.